Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM postgres
+COPY backup.backup /docker-entrypoint-initdb.d/
+FROM ruby:2.2.6
```

Comment:
Wait is that right?

Reply:
Actually, I'm not sure which version is correct to be set.